pkg/certificates/reconciler/certificates.go

```diff
@@ -146,21 +146,21 @@ func (r *reconciler) ReconcileKind(ctx context.Context, secret *corev1.Secret) p
 
 // All sans provided are required to be lower case
 func parseAndValidateSecret(secret *corev1.Secret, caCert []byte, sans ...string) (*x509.Certificate, *rsa.PrivateKey, error) {
-	certBytes, ok := secret.Data[certificates.SecretCertKey]
+	certBytes, ok := secret.Data[certificates.CertName]
 	if !ok {
-		return nil, nil, fmt.Errorf("missing cert bytes")
+		return nil, nil, fmt.Errorf("missing cert bytes in %q", certificates.CertName)
```

Comment:
can we also add the secret name + namespace?

Reply:
Yes, current prints the logs like:

```
{"severity":"INFO","timestamp":"2023-09-21T06:09:48.57517051Z","logger":"controller","caller":"reconciler/certificates.go:80","message":"Updating secret \"knative-serving\":\"knative-serving-certs\"","commit":"28effd9","knative.dev/pod":"controller-b98b48949-88w7f"}
{"severity":"INFO","timestamp":"2023-09-21T06:09:48.575482762Z","logger":"controller","caller":"reconciler/certificates.go:124","message":"Secret invalid: missing ca cert bytes","commit":"28effd9","knative.dev/pod":"controller-b98b48949-88w7f"}
```

So we should add it. I added the log in L124 above with the common `namespace/name` pattern.